aimmo-game/simulation/avatar/avatar_wrapper.py

```diff
@@ -57,14 +22,18 @@ def die(self, respawn_location):
     def add_event(self, event):
         self.events.append(event)
 
-    def set_code(self, code):
-        if self.code != code:
-            self.code = code
-            try:
-                exec(code)
-                self.avatar = Avatar()
-            except Exception as ex:
-                raise UserCodeException("Exception in user code", ex)
+    def serialise(self):
+        return {
+            'events': [
+            #    {
+            #        'event_name': event.__class__.__name__.lower(),
```

Comment:
I'm not sure what's happening with events right now. I think we probably will want them, in some form or another